audit/format.go

```diff
@@ -293,14 +295,15 @@ func (f *AuditFormatter) FormatResponse(ctx context.Context, w io.Writer, config
 		},
 
 		Response: &AuditResponse{
-			MountType: req.MountType,
-			Auth:      respAuth,
```

Comment:
only question i have is, do we need this field in both the request and response blocks of the response audit log ? Since it feels like having it in just one place should be good enough, but I do see we do this duplication for other fields like mountType and headers, so maybe there's a rationale i'm missing here.

Reply:
Yes, we duplicate these because there may be cases where a request occurs but the response doesn't due to error or what have you.

Reply:
aah, gotcha!